ansible/configs/ibm-rhoic/software.yml

```diff
@@ -108,47 +114,56 @@
       uri:
         url: "{{ ibm_cloud_api_container_v2_url }}/getCluster?cluster={{ rhoic_cluster_id }}"
         method: GET
-        status_code: 200,404,500
+        status_code: 200
         headers:
           Authorization: Bearer {{ ibm_access_token }}
       register: r_existing_rhoic
       tags: retrieve-rhoic
       until: r_existing_rhoic.json.state is defined and r_existing_rhoic.json.state == "normal"
       retries: 30
       delay: 120
-    - name: Download cluster client configuration
-      uri:
-        url: "{{ ibm_cloud_api_container_v1_url }}/{{ rhoic_cluster_id }}/config/admin"
-        method: GET
-        dest: "{{ output_dir }}/{{ rhoic_cluster_id }}.zip"
-        headers:
-          Authorization: Bearer {{ ibm_access_token }}
-      tags: get-certificates
-    - name: Unarchiving the client config zip file
-      unarchive:
-        src: "{{ output_dir }}/{{ rhoic_cluster_id }}.zip"
-        dest: "{{ output_dir }}"
-    - name: Retrieve file names
-      find:
-        paths: "{{ output_dir }}/kubeConfig000000000"
-        recurse: true
-        file_type: file
-      register: r_files
     - name: Displaying RHOIC cluster details
       debug:
         var: r_existing_rhoic
         verbosity: 2
+    ## Sending details of the cluster to the client
     - name: Passing along the cluster info
       agnosticd_user_info:
         msg: "{{ item }}"
       loop:
-        - "RHOIC RHPDS Lab Guide (README IMPORTANT!): https://red.ht/ROKSRHPDS"
+        - "The information below can be used to access your Red Hat OpenShift on IBM Cloud Sandbox environment."
+        - "IBM Cloud User: {{ email }}"
         - "OpenShift Admin Console: {{ r_existing_rhoic.json.masterURL }}"
         - "IBM Cloud Console: https://cloud.ibm.com/"
-    - name: Passing along the cluster admin
-      agnosticd_user_info:
-        msg: "=== {{ item.split('/')[-1] }} ===\n{{ lookup('file', item) }}"
-      with_items: "{{ r_files.files | map(attribute='path') | list }}"
+        - "RHOIC RHPDS Lab Guide (README IMPORTANT!): https://red.ht/ROKSRHPDS"
+
+# start of output for the kubectl/oc config and certs
```

Comment:
Please remove commented out code since it is not needed.

Reply:
done, merging now